### DISCUSSION TITLE
Pacman is eating himself!

### DISCUSSION THREAD

**Discussion by cherrington-maxwell** [Napster2210/ngx-spinner#271] [Q&A]
I have a weird problem.  I am using Angular 17 with ngx-spinner 16.0.2
I've got a busy service to detect concurrent calls and only show and hide the spinner once.  I've confirmed this by writing out to the console "busy" or "idle" and I only see it logged once.
Oddly though, I am seeing a duplicate of the pacman animation.  One overtop of the other; one slightly smaller than the other.
I have the ngx-spinner only defined in my app.component.html like this:

```
<ngx-spinner bdColor = "rgba(0, 0, 0, 0.8)" size = "large" color = "#fff" type = "pacman" [fullScreen] = "true">
    <p style="color: white" > Loading... </p>
</ngx-spinner>
```
![bad-pacman](https://github.com/Napster2210/ngx-spinner/assets/102751920/414fa7af-1370-4f1f-ae8d-b88990489ec0)

Any idea why it would be doing this?

**Comment by cherrington-maxwell** [comment]
Interestingly enough, I can replicate by going to https://napster2210.github.io/ngx-spinner/
![2023-12-12 14_46_58-NgxSpinner _ Demo](https://github.com/Napster2210/ngx-spinner/assets/102751920/13450451-d71f-4d36-a7ae-389895fc968e)

**Reply by revati9navgire** [reply]
I am getting the same issue the layered pacman design after migrating application to angular 18. I am using angular 18 and ngx-spinner 18.0.0 version type pacman. Is there any solution for this

**Comment by Napster2210** [comment]
This was CSS issue and it is resolved now
